news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,283 @@
+
+But originals are not the only highlights of the pool! As we all know by now, the mini-TB gives players a taste of how an actual tiebreaker is typically designed, both in terms of structure and difficulty. [mitei - Phosphor](https://osu.ppy.sh/beatmapsets/2317893#mania/4964059) (HB3) by [U1d](https://osu.ppy.sh/users/10125072) has it all! Aesthetically pleasing patterns are present throughout the map while covering a wide range of skill sets required to play these patterns effectively. Beyond these challenges, it also offered an enjoyable experience not only for the players alone but also for the mappool playtesters. A 0% ban rate during the weekend is a testament to that.
+
+! EXTRA MAP TO HIGHLIGHT HERE !
+
+---
+
+[![](/wiki/shared/news/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap/FOXGLOVE_banner.jpg)](https://osu.ppy.sh/beatmapsets/2318030#mania/4964416)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/433/Songs/Xyris_-_FOXGLOVE.mp3">
+    </audio>
+</div>
+
+Following suit from last week, we've got a couple new original tracks lined up! The second one to showcase was [FOXGLOVE](https://osu.ppy.sh/beatmapsets/2318030#mania/4964416) (HB1) produced by [Xyris](https://osu.ppy.sh/beatmaps/artists/433) with the backup of [Critical_Star](https://osu.ppy.sh/users/3793196) as the maestro behind the map. A fairly generic hybrid as the category would suggest. Not to be confused with simple however given the map's tricky nature. With fast-paced chordstreams to the rhythm of the song's very upbeat melody and less-than-friendlier releases, this beatmap is anything but simple.
+
+[![](/wiki/shared/news/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap/Ruin_in_Cascade_banner.jpg)](https://osu.ppy.sh/beatmapsets/2318032#mania/4964418)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/367/Songs/kanemiko_-_Ruin_in_Cascade.mp3">
+    </audio>
+</div>
+
+Our second and last original track of the week was [Ruin in Cascade](https://osu.ppy.sh/beatmapsets/2318032#mania/4964418) by [kanemiko](https://osu.ppy.sh/beatmaps/artists/367). Keeping up with the tradition of featuring exciting new tracks on the tiebreaker, this year's 7K MWC was no exception! A massive collaborative effort between [Critical_Star](https://osu.ppy.sh/users/3793196), [_Stan](https://osu.ppy.sh/users/1653229), [Maxus](https://osu.ppy.sh/users/4335785), [kaulpakka](https://osu.ppy.sh/users/7115625), [ExNeko](https://osu.ppy.sh/users/7590894), and [taba2](https://osu.ppy.sh/users/7850508).
```

Comment:
again, overused incomplete sentences

```suggestion
Our second and last original track of the week was [Ruin in Cascade](https://osu.ppy.sh/beatmapsets/2318032#mania/4964418) by [kanemiko](https://osu.ppy.sh/beatmaps/artists/367). Keeping up with the tradition of featuring exciting new tracks on the tiebreaker, this year's 7K MWC was no exception! This was a massive collaborative effort between [Critical_Star](https://osu.ppy.sh/users/3793196), [_Stan](https://osu.ppy.sh/users/1653229), [Maxus](https://osu.ppy.sh/users/4335785), [kaulpakka](https://osu.ppy.sh/users/7115625), [ExNeko](https://osu.ppy.sh/users/7590894), and [taba2](https://osu.ppy.sh/users/7850508).
```

Reply:
In my opinion, for the tone of the news post, these do seem to be fine in the grand scheme of things. Personally do not think it's that big of a deal, so long it is used sparingly for editorial purposes. Will apply the suggestion regardless.